.github/workflows/ci.yml

```
@@ -49,14 +47,18 @@ jobs:
       - uses: actions/checkout@v2
       
       #- name: install utils
-      #  run: apt-get update && apt-get install -y openssh-client postgresql-client curl
+      #  run: |
+      #    sudo apt-get update
+      #    sudo apt-get install -y openssh-client postgresql-client curl
         
       - name: db setup
         uses: docker://pecan/db:latest
 
       - name: create account
-        uses: docker://pecan/bety:latest
-        args: user carya illinois "Carya Demo User" carya@example.com 1 1
+        uses: docker://pecan/bety:develop
+        with:
+          # entrypoint: /home/bety/script/betyuser.sh
+          args: carya illinois "Carya Demo User" carya@example.com 1 1
```

Based on the [error message](https://github.com/PecanProject/rpecanapi/runs/1303240970#step:7:10), I think this is missing the target function (`user`):

```suggestion
          args: user carya illinois "Carya Demo User" carya@example.com 1 1
```